src/sentry/integrations/github/integration.py

```diff
@@ -936,7 +951,10 @@ def dispatch(self, request: HttpRequest, pipeline: Pipeline) -> HttpResponseBase
             return self.render_react_view(
                 request=request,
                 pipeline_name="githubInstallationSelect",
-                props={"installation_info": installation_info},
+                props={
+                    "installation_info": installation_info,
```

Comment:
if they don't have multi org, should we still be passing the full list of installations to the FE?

Reply:
i think we should also error if somebody tries to pass a chosen installation id that's not -1 and they don't have multi org

Reply:
For the passing the full list comment, I think it's useful to keep in case we want to do the per org upsell in the dropdown in the UI.